extensions/amp-sidebar/0.1/amp-sidebar.js

```diff
@@ -373,6 +381,10 @@ export class AmpSidebar extends AMP.BaseElement {
       this.vsync_.mutate(() => {
         toggle(this.element, /* display */false);
         this.schedulePause(this.getRealChildren());
+        // Return focus to source element if applicable
+        if (this.sourceElement_) {
```

Comment:
when you merge my PR, let's keep the trigger of the close action "after" this line so that if they happen to have a custom focus target using `on:sidebarClose`, we won't override it.